Sources/SpeziScheduler/Notifications/SchedulerNotifications.swift

```diff
@@ -223,9 +224,9 @@ public final class SchedulerNotifications {
             logger.debug("Background fetch is not enabled. Skipping registering background task for notification scheduling.")
             #endif
         }
-
-        _Concurrency.Task { @MainActor in
-            await checkForInitialScheduling(scheduler: scheduler)
+        _Concurrency.Task.detached {
+            try await _Concurrency.Task.sleep(for: .seconds(4))
```

Comment:
To fully understand this, we should document the reason for the sleep here. And does it make sense to make this shorter to ensure that we don't run into any elements where they are not scheduled correctly?

Reply:
i think we might be able to shorten this actually, or maybe even remove it entirely; the 4 second delay is a leftover from a previous version of the PR. i'll have a look

Reply:
Thank you!